### PR TITLE
16357: Add lat and lon to institutions

### PR DESCRIPTION
**Pull request by elder-joseph**
## Description
As a developer, I need to include the latitude and longitude in version generation so that it can be used for any forthcoming mapping / near me functionality.

#### AC1 Analysis:
When testing with the local set of sample CSVs, only 9599 of the 69343 institutions have latitude values and 9600 of the same 69343 have longitude values.

Note: Longitude is supplied as `longitud` from the `ipeds_hd` csv file.

[Originating Issue](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/16357)

## Testing done
CI tests pass locally.

## Screenshots
N/A

## Acceptance criteria
- [x] Analysis of data coverage of institutions with/without latitude and/or longitude is documented on this story.
- [x] Institution Latitude is:
a. included in version generation.
b. included in the institution export.
c. available to be used by the front end of the Comparison Tool
- [x] Institution Longitude is:
a. included in version generation.
b. included in the institution export.
c. available to be used by the front end of the Comparison Tool

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: app/models/institution.rb [hunk 1]
```diff
@@ -163,7 +163,9 @@ class Institution < ImportableRecord
     'complies_with_sec_103' => { column: :complies_with_sec_103, converter: BooleanConverter },
     'solely_requires_coe' => { column: :solely_requires_coe, converter: BooleanConverter },
     'requires_coe_and_criteria' => { column: :requires_coe_and_criteria, converter: BooleanConverter },
-    'poo_status' => { column: :poo_status, converter: BaseConverter }
+    'poo_status' => { column: :poo_status, converter: BaseConverter },
+    'latitude' => { column: :latitude, converter: NumberConverter },
+    'longitud' => { column: :longitud, converter: NumberConverter }
```
Comment (zurbergram) [inline app/models/institution.rb:168]:
```suggestion
    'longitude' => { column: :longitud, converter: NumberConverter }
```

File: app/models/institution_builder.rb [hunk 2]
```diff
@@ -310,6 +311,18 @@ def self.add_scorecard(version_id)
 
     Institution.connection.update(str)
   end
+  
+  def self.update_lat_lon_from_scorecard(version_id)
```
Comment (zurbergram) [inline app/models/institution_builder.rb:315]: This is not needed as column is set from scorecard at https://github.com/department-of-veterans-affairs/gibct-data-service/blob/master/app/models/institution_builder.rb#L303

File: app/models/ipeds_hd.rb [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class IpedsHd < ImportableRecord
-  COLS_USED_IN_INSTITUTION = %i[vet_tuition_policy_url f1sysnam f1syscod].freeze
+  COLS_USED_IN_INSTITUTION = %i[vet_tuition_policy_url f1sysnam f1syscod latitude].freeze
```
Comment (zurbergram) [inline app/models/ipeds_hd.rb:4]: This is not needed since you are manually adding the column in `institution_builder`

**Review by zurbergram — CHANGES_REQUESTED** [review]

**Review by zurbergram — APPROVED** [review]

**Review by sonntag-adam — APPROVED** [review]

**Review by thilton-oddball — CHANGES_REQUESTED** [review]
So it looks like the migrations were broken out into their own PR (https://github.com/department-of-veterans-affairs/gibct-data-service/pull/775) which is good, but those migrations should be removed from this PR. Otherwise, everything else looks good.

**Comment by elder-joseph** [comment]
@thilton-oddball no problem, I will address that and remove them from this PR.

**Comment by elder-joseph** [comment]
@thilton-oddball I merged #775 into master then merged master into this branch to effectively remove those lines from this PR.

**Review by thilton-oddball — APPROVED** [review]
Looks good!

**Pull request merged**